lib/constants/scopes.js

```diff
@@ -18,17 +20,29 @@ export const XAPI_STATEMENTS_WRITE = 'statements/write';
 export const XAPI_STATEMENTS_READ_MINE = 'statements/read/mine';
 export const XAPI_STATE_ALL = 'state';
 export const XAPI_PROFILE_ALL = 'profile';
+export const XAPI_STATEMENTS_DELETE = 'statements/delete';
 
 export const XAPI_SCOPES = {
   [XAPI_ALL]: 'All',
   [XAPI_READ]: 'Read all',
   [XAPI_STATEMENTS_READ]: 'Read all statements',
   [XAPI_STATEMENTS_WRITE]: 'Write statements (must be used with a read scope)',
   [XAPI_STATEMENTS_READ_MINE]: 'Read my statements',
+  [XAPI_STATEMENTS_DELETE]: 'Delete statements (not covered by all scope)',
   [XAPI_STATE_ALL]: 'Access state',
   [XAPI_PROFILE_ALL]: 'Access profiles',
 };
 
+export const getXapiScopes = (
+  deleteStatementsEnabled = get(process.env, 'ENABLE_SINGLE_STATEMENT_DELETION', true)
```

Comment:
I can't remember what webpack does in this case, but does this potentially expose our entire `process.env` since I think webpack replaces references to the `process.env`. If this is the case we need explicitly look for `process.env. ENABLE_SINGLE_STATEMENT_DELETION`.

Reply:
Not sure how that would work? There is no entry point for dotenv config in the client context.

Reply:
@ht2 any idea whether Webpack injects this?

Reply:
I'm not sure guys, is this used client side, and does it pull the correct value from the .env when you've tested it? Presumably this is built and read from the .env when the client server is started, but Chris is probably right that there isn't an entry point

Reply:
I think we just need to log out the `process.env` then and see what happens.

Reply:
Ok I think this is fine, I've added the var to the webpack build.